app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt

```diff
@@ -256,6 +256,65 @@ class HelpFragmentTest {
     }
   }
 
+  @Test
+  @Config(qualifiers = "sw600dp")
+  fun testHelpFragment_defaultConfig_displaysFAQList() {
+    launch<HelpActivity>(
+      createHelpActivityIntent(
+        internalProfileId = 0,
+        isFromNavigationDrawer = true
+      )
+    ).use {
+      onView(withId(R.id.faq_fragment_recycler_view)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  @Config(qualifiers = "sw600dp")
+  fun testHelpFragment_selectFAQs_displaysFAQList() {
+    launch<HelpActivity>(
+      createHelpActivityIntent(
+        internalProfileId = 0,
+        isFromNavigationDrawer = true
+      )
+    ).use {
+      onView(
+        atPosition(
+          recyclerViewId = R.id.help_fragment_recycler_view,
+          position = 0
+        )
+      ).perform(click())
+      onView(withId(R.id.faq_fragment_recycler_view)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  @Config(qualifiers = "sw600dp")
+  fun testHelpFragment_selectThirdPartyDependencies_displaysThirdPartyDependencyList() {
+    launch<HelpActivity>(
+      createHelpActivityIntent(
+        internalProfileId = 0,
+        isFromNavigationDrawer = true
+      )
+    ).use {
+      onView(withId(R.id.help_fragment_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(1)
+      )
+      onView(
+        atPosition(
+          recyclerViewId = R.id.help_fragment_recycler_view,
+          position = 1
+        )
+      ).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.third_party_dependency_list_fragment_recycler_view)).check(
+        matches(
+          isDisplayed()
+        )
+      )
+    }
+  }
```

Comment:
Add configuration related tests too, meaning if a fragment is loaded on config change it stays the same.
Also add test for checking content description of `back arrow`
Another test to showcase that back-arrow and heading is visible in tablet and not visible in mobile.

Reply:
Added more test cases as suggested, PTAL.